### PR TITLE
Feat/12 company scraper

### LINKED ISSUE #12: Implement company name scrapper
Potentially already solved by #11 
@alexboii I leave it to your discretion if you consider to polish more that scrapper unit.

### PR DESCRIPTION
**Pull request by alexboii**
Closes #12.

Adds company scraper

### CODE REVIEW

File: src/ScrapingFunctions.re [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
-    tends to be the posted date of the job application. Subsequent matches
-    are often posted dates for other offers being advertized.*/
-let postedDateRegex = [%bs.re "/(\d+)[\s]+day[s]?[\s]+ago/im"];
-
 let scriptUrl = "document.URL";
 
 /** TODO: Need to find some good logic for this,
 Right now it's here to demonstrate erro validation */
-let scriptCompany = "document.h1";
+let scriptCompany = "document.body.innerHTML";
```
Comment (CamiloGarciaLaRotta) [inline src/ScrapingFunctions.re:5]: You can use `scriptBody` from a couple of lines below

**Comment by CamiloGarciaLaRotta** [comment]
I like the refactoring of the Regexes to `Constants` and the new `Services` module.
Do you think I should extract the date manipulation fns to `Services`?

With respect to the merge conflicts:
- I see its just a matter of both of us adding new dependencies to `package.json`so the lock file shits itself. But I see no major conflict between our new dependencies.
- for `ScrapingFunctions` my only change was adding the type to certain fns I tested and extract `formatDate` into its own fn.

If you have trouble I can pull your branch and solve the merge conflicts

**Review by CamiloGarciaLaRotta — APPROVED** [review]
bacano

**Comment by CamiloGarciaLaRotta** [comment]
I suggest rebasing your branch on top of master again instead of doing a `git pull`

**Comment by alexboii** [comment]
nah, the services should be kept for API calls, imo 

I'll fix the merge conflict

how do I rebase? is it this? https://stackoverflow.com/questions/7297379/how-do-you-rebase-the-current-branchs-changes-on-top-of-changes-being-merged-in

**Comment by CamiloGarciaLaRotta** [comment]
yep, that's the way.
I do the following
```bash
# get the latest changes from the branch you are going to rebase onto
git checkout master
git pull
# go to the PR branch and rebase
git rebase master
# because of merge conflicts, it will prompt you to solve it
# everytime you modify a file you do the following:
git add <file>
git rebase --continue
```

**Pull request merged**